bin/setup

```diff
@@ -23,12 +23,24 @@ chdir APP_ROOT do
   #   cp 'config/database.yml.sample', 'config/database.yml'
   # end
 
-  puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
```

Comment:
Why was this removed?

Reply:
Because it was moved to a npm script: It makes easier to automate the setup process on npm script.